COPYING

```diff
@@ -1,54 +1,5 @@
-tinycompress is provided under a dual BSD/LGPLv2.1 license.  When using or
-  redistributing tinycompress, you may do so under either license.
+tinycompress is provided under:
+	SPDX-License-Identifier: LGPL-2.1-only OR BSD-3-Clause
 
-BSD LICENSE
-
-tinycompress library for compress audio offload in alsa
-Copyright (c) 2011-2012, Intel Corporation
-All rights reserved.
```

Comment:
@vinodkoul you cannot remove the Intel Copyright. 
Copyright is a different concept to the license. It's fine to use SPDX for the license, the copyright needs to stay as is.

Reply:
@plbossart If you look closely, the copyright is retained in src file where IMO it is apt. Here in COPYING file it feels a copy paste mistake, hence removal.

Reply:
Some of these COPYING files only provide the license, others combine license and copyright. I don't know what the rule is and I am not a lawyer, so when in doubt I am asking you to please keep the copyright as is. It may be a mistake but that's not for me to approve.